data/src/main/java/com/microsoft/azure/kusto/data/KustoResponseResults.java

```diff
@@ -0,0 +1,95 @@
+package com.microsoft.azure.kusto.data;
+
+import com.microsoft.azure.kusto.data.exceptions.KustoServiceError;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.*;
+
+public class KustoResponseResults implements Iterator<KustoResultSetTable> {
```

Comment:
This is just "KustoOperationResult" or "KustoResultSet"

Reply:
KustoOperationResult seems better as its not a resultSet itself